vyxal/helpers.py

```diff
@@ -67,6 +67,45 @@ def uncompress(token: lexer.Token) -> Union[int, str]:
         The uncompressed token value. The return type depends on the
         token type.
     """
+    if token.name == lexer.TokenType.COMPRESSED_STRING:
+        return uncompress_str(token.value)
+    if token.name == lexer.TokenType.COMPRESSED_NUMBER:
+        return uncompress_num(token.value)
 
-    # TODO: Implement the compression stuff
     return token.value
+
+
+def uncompress_str(string: str) -> str:
+    """
+    Uncompress a string.
+
+    Parameters
+    ----------
+    string: str
+        The string to uncompress.
+
+    Returns
+    -------
+    str
+        The uncompressed string.
+    """
+    # TODO (lyxal) Implement string (un)compression
+    raise ValueError("Not implemented")
```

Comment:
raise `NotImplementedError`? (not to be confused with the special value `NotImplemented`)

Reply:
This seems trivial for something that will be removed soon

Reply:
Mehj, ok. It was just a suggestion

Reply:
@Lyxal Maybe, but it's good to know for the future.